### ISSUE TITLE
Next Meetup - #3

### ISSUE THREAD

**Issue by franzejr** [knowbody/react-native-online-meetups#38]
This is the issue to talk about the next meetup. 🙌

Ideas for topics, check the suggested topics.

**Date/Time:** TBD

## First Talk
Speaker: @peterpme

*Title:* Practical ways of moving quickly while still retaining code quality

## Second Talk

Speaker: @GantMan 

Theme: Solidarity

## Third Talk

Speaker: @dabit3 
*Title:* AWS Amplify with React Native - new things.

**Comment by GantMan** [comment]
Sign me up!

**Comment by franzejr** [comment]
Thanks, @GantMan ! Do you mind giving us a title/topic about your talk? 

Thanks one more time!

**Comment by GantMan** [comment]
I can discuss React Native Windows perhaps!  You think people will be into that?

**Comment by franzejr** [comment]
Yeah, I think it's fine! Thanks!

**Comment by franzejr** [comment]
pinging @GantMan and @peterpme to check if you guys will keep the same talk :)

**Comment by GantMan** [comment]
O wow... so many talk ideas!  Let me choose a few and suggest some!?

**Comment by franzejr** [comment]
@GantMan I think you have chosen last year React Native on Windows. Feel free. :)

**Comment by GantMan** [comment]
I would like to talk about Solidarity :)

**Comment by dabit3** [comment]
I've been working on something that will be released soon, by August 27, having to do with AWS Amplify with React Native, I'd love to talk about that.

**Comment by franzejr** [comment]
Ok, @GantMan @dabit3!

I would like to talk about a theme that I've been trying to push forward: *(Try to) Avoid React Native development being a nightmare. *

Basically a series of problems I faced and how I am trying to solve it. If @peterpme will not give the talk, I can give a talk around this theme.

**Comment by dabit3** [comment]
I like the idea but the wording seems pretty harsh & negative for people who may be just getting in to react native, maybe calling it "How to make React Native Development Easier" or "How to Tackle Common Issues in React Native' or something along those lines sounds more friendly.

**Comment by franzejr** [comment]
Hey @dabit3 I completely see your point and I thought about that! My point here is just because I wanted to avoid people having headaches on React Native Development and I would like to point it out. Because I think it can be a nightmare if you don't do the things in the correct way. 

Your point is totally valid, but I just wanted to stress out on the words just to make clear this may be a nightmare. I appreciate your concern! 😄

**Comment by peterpme** [comment]
:wave: Excited to see this being resurrected!

I agree with Nader about "React Native Nightmare". I think the words are pretty harsh. Programming anything is a nightmare =P

**Comment by franzejr** [comment]
Programming is not a nightmare and never will be, but it can be a nightmare if you do it in the wrong way and also you need to deliver software in a tiny period of time.

Do you want to give your talk, @peterpme ?

**Comment by GantMan** [comment]
you should try pair programming with me, that's a nightmare 🤦‍♂️

**Comment by franzejr** [comment]
Let's pair @GantMan ❤️

**Comment by peterpme** [comment]
Yeah I'm down to give my talk! I will do a different variation of:

`Practical ways of moving quickly while still retaining code quality `

**Comment by franzejr** [comment]
Thanks, everyone! So, we have three talks for this next remote meetup:

## First Talk
Speaker: @peterpme

**Title:** Practical ways of moving quickly while still retaining code quality

## Second Talk

Speaker: @GantMan 

**Theme:** Solidarity

## Third Talk

Speaker: @dabit3 
**Title:** AWS Amplify with React Native - new things.


We need to define a DateTime now :)

**Comment by peterpme** [comment]
Hello! I would like to chat early / mid September if possible. I will demo our new tool :)
